source/opt/loop_descriptor.cpp

```diff
@@ -33,12 +33,17 @@ Loop::Loop(IRContext* context, opt::DominatorAnalysis* dom_analysis,
       loop_continue_(continue_target),
       loop_merge_(merge_target),
       loop_preheader_(nullptr),
-      parent_(nullptr) {
+      parent_(nullptr),
+      ir_context_(context),
+      dom_analysis_(dom_analysis),
+      induction_variable_(nullptr),
+      iterations_(0),
+      could_find_num_iterations_(true),
+      loop_control_unroll_hint_(true) {
```

Comment:
That's bound to a flag that have 5 possible values, not compatible with a boolean.

Should default to None